src/App.tsx

```diff
@@ -681,6 +768,29 @@ const MapDisplay = (
                       (100 / (zoomLevel || 100))
                     )
 
+                    // Road Click
+                    let road = -1
+
+                    if (clickX >= 45 && clickX <= 55) {
+                      if (clickY < 50) {
+                        road = 2
+                      } else {
+                        road = 0
+                      }
+                    } if (clickY >= 45 && clickY <= 55) {
+                      if (clickX < 50) {
+                        road = 1
+                      } else {
+                        road = 3
+                      }
+                    }
+
+                    if (road !== -1) {
+                      const roads = map.getRoadPoints(x, y, road)
+                      setDebugRoads(roads)
+                    } else {
```

Comment:
This else and the additional indenting with it could be avoided by using the if statement as a kind of guard statement. So add return after the setDebugRoads and get rid of the else.

Also should we set the debug octants to an empty array when clicking on a road? And also other way around: Set the debugRoads empty when clicking the octants.